pages/docs/balance/api-reference/open-banking/create-reauth-exchange-session.mdx

```diff
@@ -0,0 +1,107 @@
+---
+layout: guides
+product: balance
+title: "Create a re-auth exchange session"
+guide:
+  step: 2
+meta:
+  title: "Dwolla API Reference | Open Banking: Create Re-auth Exchange Session"
+  description: "Create a re-auth exchange session."
+---
+
+# Create a re-auth exchange session
+
+This endpoint allows you to initiate a new exchange session to refresh a user's bank account connection when their existing authorization is no longer valid. This is typically required when Dwolla's API returns an `UpdateCredentials` error on a [bank balance check](https://developers.dwolla.com/docs/balance/api-reference/open-banking/retrieve-bank-balance), indicating that the user needs to re-authenticate with their bank.
```

Comment:
Everything looks good to me! 

Just adding a note, we should likely add the `UpdateCredentials`  error in the Balance check reference as well under the table "[HTTP status and error codes](https://developers.dwolla.com/docs/balance/api-reference/open-banking/retrieve-bank-balance)".